osidb/tests/test_flaw.py

```diff
@@ -1427,6 +1443,62 @@ def test_validate_allowed_source(self):
         with pytest.raises(ValidationError, match=error_msg):
             FlawFactory(source=FlawSource.ASF)
 
+    def test_validate_article_link(self):
+        """
+        Tests that an article link not starting with https://accesss.redhat.com/
+        raises an alert.
+        """
+        flawreference = FlawReferenceFactory(
+            type=FlawReference.FlawReferenceType.ARTICLE,
+            url="https://httpd.apache.org/link123",
+        )
+        assert "wrong_article_link" in flawreference._alerts
```

Comment:
you don't test the opposite case, in which the article link *does* start with `https://access.redhat.com/`

also, there's a typo in the docstring `accesss` vs `access`

Reply:
P.S.: I see that you do use `access.redhat.com` in the tests below, but never explicitly check that the alerts are cleared/empty